<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>personForm</class>
 <widget class="QWidget" name="personForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>380</width>
    <height>460</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>380</width>
    <height>460</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>380</width>
    <height>460</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>15</x>
     <y>0</y>
     <width>350</width>
     <height>150</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>350</width>
     <height>150</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>350</width>
     <height>150</height>
    </size>
   </property>
   <widget class="QPushButton" name="btn_sign">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>90</y>
      <width>90</width>
      <height>35</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QWidget" name="dynamicWidget" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>10</y>
      <width>130</width>
      <height>71</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>130</width>
      <height>71</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>130</width>
      <height>71</height>
     </size>
    </property>
    <widget class="QPushButton" name="btn_dynamic">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>10</y>
       <width>30</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Microsoft YaHei UI</family>
      </font>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="text">
      <string>动态</string>
     </property>
    </widget>
    <widget class="QLabel" name="dynamic_label">
     <property name="geometry">
      <rect>
       <x>45</x>
       <y>0</y>
       <width>40</width>
       <height>45</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="fansWidget" native="true">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>10</y>
      <width>130</width>
      <height>71</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>130</width>
      <height>71</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>130</width>
      <height>71</height>
     </size>
    </property>
    <widget class="QPushButton" name="btn_fans">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>10</y>
       <width>30</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Microsoft YaHei UI</family>
      </font>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="text">
      <string>粉丝</string>
     </property>
    </widget>
    <widget class="QLabel" name="fans_label">
     <property name="geometry">
      <rect>
       <x>45</x>
       <y>0</y>
       <width>40</width>
       <height>45</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="attentionWidget" native="true">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>10</y>
      <width>90</width>
      <height>71</height>
     </rect>
    </property>
    <widget class="QPushButton" name="btn_attention">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>10</y>
       <width>30</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Microsoft YaHei UI</family>
      </font>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="text">
      <string>关注</string>
     </property>
    </widget>
    <widget class="QLabel" name="attention_label">
     <property name="geometry">
      <rect>
       <x>25</x>
       <y>0</y>
       <width>40</width>
       <height>45</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QWidget" name="widget_5" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>150</y>
     <width>380</width>
     <height>310</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>380</width>
     <height>310</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>380</width>
     <height>310</height>
    </size>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <property name="leftMargin">
     <number>15</number>
    </property>
    <property name="topMargin">
     <number>5</number>
    </property>
    <property name="rightMargin">
     <number>15</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QWidget" name="widget_2" native="true">
      <property name="minimumSize">
       <size>
        <width>350</width>
        <height>120</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>350</width>
        <height>120</height>
       </size>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QPushButton" name="btn_vipInfo">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>350</width>
           <height>35</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>350</width>
           <height>35</height>
          </size>
         </property>
         <property name="cursor">
          <cursorShape>PointingHandCursor</cursorShape>
         </property>
         <property name="contextMenuPolicy">
          <enum>Qt::DefaultContextMenu</enum>
         </property>
         <property name="text">
          <string>会员中心</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btn_levelInfo">
         <property name="minimumSize">
          <size>
           <width>350</width>
           <height>35</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>350</width>
           <height>35</height>
          </size>
         </property>
         <property name="cursor">
          <cursorShape>PointingHandCursor</cursorShape>
         </property>
         <property name="text">
          <string>等级</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btn_store">
         <property name="minimumSize">
          <size>
           <width>350</width>
           <height>35</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>350</width>
           <height>35</height>
          </size>
         </property>
         <property name="cursor">
          <cursorShape>PointingHandCursor</cursorShape>
         </property>
         <property name="text">
          <string>商城</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="widget_3" native="true">
      <property name="minimumSize">
       <size>
        <width>350</width>
        <height>120</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>350</width>
        <height>120</height>
       </size>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QPushButton" name="btn_personInfo">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>350</width>
           <height>35</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>350</width>
           <height>35</height>
          </size>
         </property>
         <property name="cursor">
          <cursorShape>PointingHandCursor</cursorShape>
         </property>
         <property name="text">
          <string>个人信息设置</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btn_bind">
         <property name="minimumSize">
          <size>
           <width>350</width>
           <height>35</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>350</width>
           <height>35</height>
          </size>
         </property>
         <property name="cursor">
          <cursorShape>PointingHandCursor</cursorShape>
         </property>
         <property name="text">
          <string>绑定社交账号</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btn_service">
         <property name="minimumSize">
          <size>
           <width>350</width>
           <height>35</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>350</width>
           <height>35</height>
          </size>
         </property>
         <property name="cursor">
          <cursorShape>PointingHandCursor</cursorShape>
         </property>
         <property name="text">
          <string>我的客服</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="widget_4" native="true">
      <property name="minimumSize">
       <size>
        <width>350</width>
        <height>70</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>350</width>
        <height>70</height>
       </size>
      </property>
      <widget class="QPushButton" name="btn_logout">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>10</y>
         <width>350</width>
         <height>50</height>
        </rect>
       </property>
       <property name="minimumSize">
        <size>
         <width>350</width>
         <height>50</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>350</width>
         <height>50</height>
        </size>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="text">
        <string>退出登录</string>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
